tests/integration/conftest.py

```diff
@@ -145,14 +148,20 @@ def populate_nautobot_object_types(nb_api, devicetype_library_repo_dirpath):
 
     # add device roles
     for device_role_name in DEVICE_ROLE_NAMES:
-        nb_api.dcim.device_roles.create(name=device_role_name, slug=device_role_name.lower().replace(" ", "-"))
+        nb_api.extras.roles.create(
+            name=device_role_name, slug=device_role_name.lower().replace(" ", "-"), content_types=["dcim.device"]
```

Comment:
Shouldn't need a `slug` in 2.0.
```suggestion
            name=device_role_name, content_types=["dcim.device"]
```

Reply:
OK, I need to revisit all tests and remove it then.